tests/ui/simd/shuffle.rs

```diff
@@ -6,15 +6,20 @@
 #![allow(incomplete_features)]
 #![feature(adt_const_params)]
 
+use std::marker::ConstParamTy;
+
 extern "rust-intrinsic" {
     fn simd_shuffle<T, I, U>(a: T, b: T, i: I) -> U;
 }
```

Comment:
Hm, can't we just, like, import this from core::intrinsics now?

Reply:
Yeah, this test is older than that.